ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
         targetSdkVersion 28
         versionCode 6070000
         versionName stringVersion
-        versionNameSuffix "-early1"
+        versionNameSuffix "-ga-rc1"
```

Comment:
we could omit the suffix for GA RC build from adapter gradle , ideally we would have this suffix blank for GA
we can ensure this suffix is there on TestApp for QA to differentiate

Reply:
yes, when release GA version, we should remove versionNameSuffix property.